deploy/src/multi-account.ts

```diff
@@ -5,26 +5,39 @@ import { readFileSync } from "fs";
 
 const parse = require("csv-parse/lib/sync");
 
+function parseCredentials(path: string) {
+  try {
+    const csv = parse(readFileSync(path).toString(), { columns: true })[0];
+
+    const credentials = {
+      accessKeyId: csv["Access key ID"],
+      secretAccessKey: csv["Secret access key"]
+    };
```

Comment:
Nice! This actually handles both of the formats, since the column names are the same for both :)